2022/kand/d2-1.py

```diff
@@ -0,0 +1,36 @@
+def t(s):
+    if s in ('A', 'X'):
+        return 'R'
+    elif s in ('B', 'Y'):
+        return 'P'
+    else:
+        return 'S'
+
+with open('d2-1.input') as f:
+    total = 0
+    for line in f:
+        a, b = line.replace('\n', '').split(' ')
+
+        ta = t(a)
+        tb = t(b)
+
+        print(ta, tb)
+        roundScore = 3 if ta == tb else 0
+
+        if tb == 'R':
```

Comment:
I like how you solved it programatically. i was sitting here like theres only N outcomes of this game so I'll just hardcode it all. see #5  LOL.

Reply:
Yea my code golf-ed solution for part 2 ended up there pretty much. See `d2-2-2.py` below.